pkg/util/pki/csr.go

```diff
@@ -57,6 +58,22 @@ func DNSNamesForCertificate(crt *v1alpha1.Certificate) []string {
 	return crt.Spec.DNSNames
 }
 
+func IPAddressesForCertificate(crt *v1alpha1.Certificate) []net.IP {
+	var ipAddresses []net.IP
+	for _, ip := range IPAddressesNameForCertificate(crt) {
+		ipAddresses = append(ipAddresses, net.ParseIP(ip))
+	}
+	return ipAddresses
+}
+
+func IPAddressesNameForCertificate(crt *v1alpha1.Certificate) []string {
```

Comment:
`IPAddressesForCertificate` 😄

Reply:
I already defined a function `IPAddressesForCertificate` that returns an array of `[]net.IP`

Reply:
I think this function can probably be removed altogether - `crt.Spec.IPAddresses` is already a `[]string`, and IMO we shouldn't be calling `removeDuplicates` on this returned data as duplicate entries are more of a validation error.